src/js/themes/base.d.ts

```diff
@@ -729,7 +729,10 @@ export interface ThemeType {
       container?: BoxProps;
       icon?: any;
     };
-    label?: TextProps;
+    label?: {
```

Comment:
According to the changed file, `label` used to accept all types of Text props, any reason we are singling out only two options for props now? was the implementation wrong earlier?

Reply:
You're right, all TextProps is fine, but we'd also like to allow `requiredIndicator` in addition. Looking into how to do that in the declaration file.

Reply:
Done. Created `FormFieldLabelType` interface that extends TextProps and adds requiredIndicator.